android/res/values-zh/strings.xml

```diff
@@ -141,15 +141,11 @@
 	<string name="cant_change_this_setting">抱歉，地图存储设置目前被禁用</string>
 	<!-- Alert message that downloading is in progress. -->
 	<string name="downloading_is_active">地图下载进行中。</string>
-	<!-- Share my position using SMS, %1$@ contains om:// and %2$@ https://omaps.app link WITHOUT NAME. @NOTE non-ascii symbols in the link will result in max 70 characters SMS instead of 140. -->
-	<string name="my_position_share_sms">嘿，在 Organic Maps 上查看我目前的位置吧!%1$s 或%2$s 未安装离线地图？在此下载: https://omaps.app/get</string>
-	<!-- Subject for emailed bookmark -->
-	<string name="bookmark_share_email_subject">嘿，在 Organic Maps 上查看我标注的图钉吧！</string>
-	<!-- Subject for emailed position -->
-	<string name="my_position_share_email_subject">嗨，到 Organic Maps 查看我的当前位置！</string>
-	<!-- Share my position using EMail, %1$@ is om:// and %2$@ is https://omaps.app link WITHOUT NAME -->
-	<string name="my_position_share_email">嗨，\n\n我现在在这:%1$s。点击此链 接%2$s或此链接 %3$s来查看在地图上的位置。\n\n谢谢。</string>
-	<!-- Share button text which opens menu with more buttons, like Message, EMail, Facebook etc. -->
+	<!-- Subject for emailed map point / poi / bookmark coordinates -->
+	<string name="share_coords_subject_default">嘿，在 Organic Maps 上查看我标注的图钉吧！</string>
+	<!-- Text preceding shared "my position" coordinates -->
+	<string name="share_my_position">嗨，到 Organic Maps 查看我的当前位置</string>
```

Comment:
Sorry, do you already changed `share_my_position`'s English string to `I am here on Organic Maps`?

`share_my_position`'s new zh translation is for old English string `Hi, check out my current location in Organic Maps!`, which in `my_position_share_sms`.